treeherder/webapp/api/runnable_jobs.py

```diff
@@ -57,6 +76,40 @@ def list(self, request, project):
                 'state': 'runnable',
                 'result': 'runnable'})
 
+        for label, node in tc_graph.iteritems():
+            build_platform = node['task']['extra']['treeherder']['build']['platform']
+            job_type_name = node['task']['metadata']['name']
+
+            # Not all tasks have a group name
+            if 'groupName' in node['task']['extra']['treeherder']:
+                job_group_name = node['task']['extra']['treeherder']['groupName']
+            else:
+                job_group_name = ""
+
+            ret.append({
+                'build_platform_id': "",
```

Comment:
Yeah, I'm thinking anything with an empty string should just be omitted as a field.  If the UI needs it, then we should get a valid value.  Otherwise, it's just a waste of payload size.